File: program/find-the-determinant-of-a-matrix/FindTheDeterminantOfAMatrix.cs
using System;  
public class Determinant 
{  
    public static void Main()
   {
            int i,j;
        	int[,] arr1 = new int[10,10];
            int det=0;

	 Console.Write("Input elements in the matrix :\n");
       for(i=0;i<3;i++)
        {
            for(j=0;j<3;j++)
            {
	           Console.Write("element - [{0}],[{1}] : ",i,j);
			   arr1[i,j] = Convert.ToInt32(Console.ReadLine());
            }
        }  
	 Console.Write("The matrix is :\n");
	 for(i=0;i<3;i++)
	 {
	   for(j=0;j<3 ;j++)
	     Console.Write("{0}  ",arr1[i,j]);
	    Console.Write("\n");
	 }

  for(i=0;i<3;i++)
      det = det + (arr1[0,i]*(arr1[1,(i+1)%3]*arr1[2,(i+2)%3] - arr1[1,(i+2)%3]*arr1[2,(i+1)%3]));

  Console.Write("\nThe Determinant of the matrix is: {0}\n\n",det);
  }
}

File: program/program/find-the-determinant-of-a-matrix/FindTheDeterminantOfAMatrix.cpp
#include <iostream> 
using namespace std; 

int main()
{
    int matrix[3][3];
    int determinant = 0;

    cout << "Enter the 9 elements of matrix: " << endl;

    for(int i = 0; i < 3; i++)
        for(int j = 0; j < 3; j++)
            cin >> matrix[i][j];

    cout << "\nThe matrix is\n";

    for(int i = 0; i < 3; i++)
    {
        cout << "\n";

        for(int j = 0; j < 3; j++)
            cout << matrix[i][j] << "\t";
    }

    for(int i = 0; i < 3; i++)
        determinant = determinant + (matrix[0][i] * (matrix[1][(i+1)%3] * matrix[2][(i+2)%3] - matrix[1][(i+2)%3] * matrix[2][(i+1)%3]));

    cout << "\nDeterminant of the matrix is: " << determinant << endl;


    return 0;
}